doc/src/Howto_apip.rst

```diff
@@ -0,0 +1,200 @@
+Adaptive-precision interatomic potentials (APIP)
+================================================
+
+The :ref:`APIP package <PKG-APIP>` allows to use adaptive-precision potentials
```

Comment:
... PKG-APIP enables use of adaptive-precision potentials as described in (citation)...